src/ts/tests/circular_buffer.spec.ts

```diff
@@ -165,6 +165,20 @@ test('[UNIT] at/1', () => {
 
 
 
+test('[UNIT] fill/5', () => {
+
+  // declare a three item cb
+  const filler = new circular_buffer<number>(5);
+
+  // can accept three items
+  filler.fill(`hello`);
+
+  expect(filler.fill('hello')).toStrictEqual(["hello","hello","hello","hello","hello"]);
```

Comment:
do me a favor and drop `.toStrictEqual` to the next line, two spaces in?  when they're this long gh mis-renders them

Reply:
also you don't need line 174.  the one it's going to compare against is the one inside expect(...)